banzai/qc/header_checker.py

```diff
@@ -170,17 +191,28 @@ def check_exptime_value(self, image):
 
         """
 
-        exptime_value = image.header['EXPTIME']
-
         logging_tags = logs.image_config_to_tags(image, self.group_by_keywords)
 
-        if exptime_value < 0.0:
-            sentence = 'The header EXPTIME key got the unexpected value : negative value'
-            self.logger.error(sentence, extra=logging_tags)
+        try:
+            exptime_value = image.header['EXPTIME']
+            self.save_qc_results({"header_exptime": exptime_value}, image)
+        except:
```

Comment:
Should there be behavior here if the value doesn't exist?

Reply:
Yes. @turnerm this should put a flag in the qc database if there is no exposure time because this means the fits file was totally borked.